File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        printf("Height: ");
        height = get_int("");
    }
    while ((height <= 0) || (height > 8)) ;
    {
        for (int layer = 1; layer <= height; layer++)
        {
            for (int i = 1; i <= height; i++)
            {
                if ((i + layer) > height)
                {
                printf("#");
                }
                else
                {
                    printf(" ");
                }
            }
            printf(" ");
            for (int i = 1; i <= layer; i++)
            {
                printf("#");
            }
        printf("\n");
        }
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

height = get_int("Height: ")

while ((height > 0) and (height <= 8)) :
    for layer in range (0, height, 1):
        blank = ""
        block = ""

        for i in range(0, (height - layer), 1):
            blank += " "

        for i in range(0, layer + 1, 1):
            block += "#"

        print(blank ,block, " ", block, blank)

    height = get_int("Height: ")